packages/ui/src/components/va-sidebar/VaSidebarItem/VaSidebarItem.vue

```diff
@@ -71,6 +71,7 @@ export default defineComponent({
 
       if (isHovered.value || props.active || hasKeyboardFocus.value) {
         style.backgroundColor = backgroundColorComputed.value
+        style.transition = 'background-color 0.6s ease-in-out'
```

Comment:
```suggestion
        style.transition = 'background-color var(--va-sidebar-item-transition)'
```

Reply:
Okay. I would update my code with these suggestions. Thanks

Reply:
We also have global `--va-transition` variable.

It's even better to create
```
--va-sidebar-item-transition: var(--va-transition);
```

This way, user can change transition for ALL components in one place by changing `--va-transition` or change transition specific for sidebar by changing `--va-sidebar-item-transition`.

Sorry for not writing it in issue.